types/bun/bundle.ts

```diff
@@ -19,20 +19,15 @@ const filesToCat = (
   await file(join(import.meta.dir, "paths.txt")).text()
 ).split("\n");
 
-const text =
-  header +
-  (
-    await Promise.all(
-      filesToCat.map(
-        async (name) =>
-          "// " +
-          name +
-          "\n\n" +
-          (await file(resolve(import.meta.dir, name)).text()) +
-          "\n"
-      )
-    )
-  ).join("\n");
+const fileContents: string[] = [];
+
+for (let i = 0; i < filesToCat.length; i++) {
+  const name = filesToCat[i];
+  fileContents.push("// " + name + "\n\n" + (await file(resolve(import.meta.dir, name)).text()) + "\n");
```

Comment:
This was intentionally loading the files in parallel instead of one at a time.

Reply:
Ah, that makes a bit more sense. Should I just revert this then?

Reply:
Yeah